fontforgeexe/startui.c

```diff
@@ -391,7 +396,8 @@ static void start_splash_screen(void){
     GDrawSync(NULL);
     GDrawProcessPendingEvents(NULL);
     GDrawProcessPendingEvents(NULL);
-    splasht = GDrawRequestTimer(splashw,1000,1000,NULL);
+
+    splasht = GDrawRequestTimer(splashw,7000,1000,NULL);
```

Comment:
Not sure why the change to 7000 but it doesn't seem to cause any problems.

Reply:
Because before it was using splash_cnt >=7 to decide when to close the window, as it was doing different things on each timer (it resized on splash_cnt=1, 2 and 3), but now since we only need the timer to fire once I just made it 7000